components/02-molecules/meta/event-meta/yds-event-meta-localist.twig

```diff
@@ -168,8 +181,7 @@
             <ul {{ bem('multiple-dates', [], event_meta__base_class) }} aria-expanded="false">
               {% for event_date in event_dates %}
                 <li {{ bem('multiple-dates__date', [], event_meta__base_class) }}>
-                  {{ event_date.formatted_start_date }} {{ event_date.formatted_start_time }} to {{ event_date.formatted_end_date }} {{ event_date.formatted_end_time }}
-                </li>
+          {{ event_dates.0.formatted_start_date }} {{ event_dates.0.formatted_start_time }} - {{ event_dates.0.formatted_end_time }}                 </li>
```

Comment:
I believe this needs to mimic the above if statement you created also so that they are displaying the same.  What do you think?

Also, you are referencing `event_dates` here when you're looping through them, so you'd want `event_date` I believe.  Looking at the changed code you'd just reference them like this:  `event_date.formatted_start_date`

Reply:
So while you deleted your code in your latest push, you didn't revert to the old code that was there.  ☹️ You'll want to place back the above that's in red if the green wasn't meant to be there.